example/scp.c

```diff
@@ -145,6 +145,10 @@ int main(int argc, char *argv[])
         goto shutdown;
     }
 
+#ifdef WIN32
```

Comment:
Is this important for receiving > 2gb files?  If so, can you explain why in a comment?  If it's a generally good thing for win32, but not for >2gb support specifically, do you mind if it goes in a separate commit, and maybe even a separate pull request directly against upstream?